src/helper/hover.js

```diff
@@ -11,6 +11,7 @@ import {isGroupChild} from './group';
  * @return {paper.Item} the hovered item or null if there is none
  */
 const getHoveredItem = function (event, hitOptions, subselect) {
+    // @todo make hit test only hit painting layer
```

Comment:
?

Reply:
I added this todo because we have to set data.noHover and data.noSelect all over the place when it should just be the default for everything that's not on the painting layer. It would make the code cleaner.